tests/unit/combo/tests.html

```diff
@@ -5729,6 +5733,37 @@ <h2 id="qunit-userAgent"></h2>
                 equal($.ig.util.areSetsEqual(x, y), true, "Arrays x:" + x + " and y:" + y + " are equal (same reference)");
                 equal($.ig.util.areSetsEqual(x, x), true, "Arrays x:" + x + " and x:" + x + " are equal");
             });
+
+            test(testId_95, function() {
+
+                var data, $combo, $input;
+
+                data = [
+                    { "ID":1, "Name": "John Smith", "Age": 45 },
+                    { "ID":2, "Name": "Mary Johnson", "Age": 32 },
+                    { "ID":3, "Name": "Bob Ferguson", "Age": 27 }
+                ];
+                $combo = $('#filteringExpressions');
+                $combo.igCombo({
+                    dataSource: data,
+                    animationShowDuration: 0,
+                    animationHideDuration: 0,
+                    valueKey: "ID",
+                    textKey: "Name",
+                    filteringType: "local",

```

Comment:
No, the test should subscribe to the filtering event, perform the check inside the handler with `ui.expression`. There should be no `setTimeout`